models/models.go

```diff
@@ -12,6 +12,10 @@ type DBModel struct {
 	DeletedAt *time.Time `json:"-"`
 }
 
+type LoginValues struct{
```

Comment:
Do we really need this struct out of testing scope?

Reply:
In the initial testing phase, we were supposed to give login values to the request, as per that time, I was trying out to give values initialize from a struct. Ya, we can remove that now, as we are embedding the values for test directly into the request. Amended in the commit bf8720051cec65504a9d5c5c108cf4b11a42fbb0